AssetManagementTestProject/TestCases/US303LogoutTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿using NUnit.Framework;
+using NUnit.Framework.Internal;
+using AssetManagementTestProject.PageObj;
+using AssetManagementTestProject.TestSetup;
+
+namespace AssetManagementTestProject.TestCase;
+
+[TestFixture]
+public class US303LogoutTest : NUnitWebTestSetup
+{
+    [TestCase(Constant.ADMIN_USERNAME_HN, Constant.BASE_ADMIN_PASSWORD)]
+    [TestCase(Constant.STAFF_USER_NAME, Constant.STAFF_PASSWORD)]
+    public void LogoutSuccessfully(string userName, string password)
+    {
+        LoginPage.Login(userName, password); 
+        Asserter.AssertStringEquals(HomePage.ReturnHomePageUrl(), Constant.BASE_URL);
+
+        HomePage.SelectLogout();
+        LogoutPopup.LogOutOfPage();
+        Asserter.AssertElementsAreDisplayed(LoginPage.LoginPageElementUI());
```

Comment:
You're checking only the username text box.
Please define exact element you want to check

Reply:
???